test/index.js

```diff
@@ -16,6 +16,20 @@ describe("jshint loader", function() {
 		});
 	});
 
+	it("should find and coalesce nested .jshintrc files", function() {
+		var host = {
+			resourcePath: conf.entry,
+			addDependency: function() {}
+		};
+		loadRcConfig.call(host).should.deepEqual({
+			bitwise: true,
```

Comment:
I see now. Maybe you could achieve the same result without tweaking the root level jshintc.